### ISSUE TITLE
RunAndRunOnce Method does not detect powershell.exe entries

### ISSUE THREAD

**Issue by strassi** [last-byte/PersistenceSniper#23]
Hi,

awesome project! I found a bug though, that might be rooted in the way you reference the lolbas project, because this project is missing "powershell.exe" as a lolbin 🤔 🤣 

### Setup
```
PersistenceSniper ModuleVersion = '1.15.0'

PSVersion                      5.1.22621.2506
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.22621.2506
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Current Behavior
![image](https://github.com/last-byte/PersistenceSniper/assets/14335437/6a0a049f-e9ce-4417-8cf4-e62aae7f1589)
I created two Run Keys under the current user (Skype and SymantecEndpointProtection)

![image](https://github.com/last-byte/PersistenceSniper/assets/14335437/1618d407-80e2-43b9-bfe4-7a8d8d33243d)
I ran the Persistence Sniper as "Administrator". You can see a modified output that I used for debugging. It states, that the binary is considered safe. This output is given if the `Get-IfSafeExecutable` is true. 

![image](https://github.com/last-byte/PersistenceSniper/assets/14335437/28aae0ff-3c03-42a7-b4fc-865fe7c2bc47)

The `Get-IfSafeExecutable` has an additional LolBas check. This checks if the prop value contains a lolbas binary. The problem is, that `powershell.exe` is not in this list. Since it is a Builtin Binary and the lolbas check fails, the binary is considered safe.

![image](https://github.com/last-byte/PersistenceSniper/assets/14335437/6539f602-1d26-4b95-8842-5b8d5599a2ff)

![image](https://github.com/last-byte/PersistenceSniper/assets/14335437/86a9a22d-c2ea-4aed-b9ab-36be31e23ef0)

### Expected Behavior

The output of the PersistenceSniper should display the registry key "Skype".

### Additional Details

I get the same invalid behavior if I use the `-IncludeHighFalsePositivesChecks` switch.

### Possible solutions

- add `powershell.exe` to lolbas project
- implement `powershell.exe` in the `Get-ifLolBin` list manually
- implement a `Get-ifCmdInterpreter` with `powershell.exe`, `cmd.exe` as blacklist to always report persistence using such command line interpreters.

**Comment by last-byte** [comment]
Mmmmh... This is definitely a bug in that the function should return the presence of the Run key regardless of how safe the binary is considered to be. I'll make sure to fix it right away, thanks for pointing this out!

**Comment by last-byte** [comment]
I think the most straightforward fix is to just add powershell.exe to the lolbin list. I'm curious though as to why LOLBAS' maintainers did not include Powershell.exe to the list...

**Comment by last-byte** [comment]
Fixed it in version [1.15.1](https://github.com/last-byte/PersistenceSniper/blob/main/CHANGELOG.md#1151).
